services/movie_session.py

```diff
@@ -42,3 +42,11 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
+    if MovieSession.objects.filter(id=movie_session_id).exists():
```

Comment:
no need for extra if's and prints, just return result - we don't need to try to catch all errors in this scenario